src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -322,6 +322,9 @@ def _detect_performance_problems(data: Event, sdk_span: Any) -> List[Performance
 
     truncated_problems = detected_problems[:PERFORMANCE_GROUP_COUNT_LIMIT]
 
+    metrics.incr("performance.performance_issue.detected", len(detected_problems))
+    metrics.incr("performance.performance_issue.truncated", len(truncated_problems))
```

Comment:
Side note, I might go through all our existing metrics and clean them up / write them down since there are a lot and some will probably more useful long term.